x-pack/solutions/security/plugins/security_solution/ui_tests/parallel_tests/flyout/alert_details_url_sync.spec.ts

```diff
@@ -11,34 +11,40 @@ import { CUSTOM_QUERY_RULE } from '@kbn/scout-security/src/playwright/constants/
 const RIGHT = 'right';
 
 spaceTest.describe('Expandable flyout state sync', { tag: ['@ess', '@svlSecurity '] }, () => {
-  spaceTest.beforeEach(async ({ browserAuth, apiServices }) => {
-    await apiServices.detectionRule.deleteAll();
-    await apiServices.detectionRule.createCustomQueryRule(CUSTOM_QUERY_RULE);
+  let ruleName: string;
+  spaceTest.beforeEach(async ({ browserAuth, apiServices, scoutSpace }) => {
+    ruleName = `${CUSTOM_QUERY_RULE.name}_${scoutSpace.id}_${Date.now()}`;
+    await apiServices.detectionRule.createCustomQueryRule({ ...CUSTOM_QUERY_RULE, name: ruleName });
     await browserAuth.loginAsPlatformEngineer();
   });
 
-  spaceTest('should test flyout url sync', async ({ pageObjects }) => {
+  spaceTest.afterEach(async ({ apiServices }) => {
+    await apiServices.detectionRule.deleteAll();
+  });
+
+  spaceTest('should test flyout url sync', async ({ pageObjects, page }) => {
     await pageObjects.alertsTablePage.navigate();
```

Comment:
Question: should we `await page.waitForLoadingIndicatorHidden()` after we `navigate()` to a page?

I ask this because I saw we do wait for the loading indicator to go away after we `reload()` later in the test.

Reply:
Good question. I believe we shouldn't use waiters by default, but only when the next action stability requires it.
The reason is time execution will grow without much benefits.